templates/include/glean/custom_distribution.1.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "type": "object",
+  "properties": {
+    "range": {
```

Comment:
Is this implementation already in flight? We could do `min_bucket` and `max_bucket` properties instead of an array here.

Reply:
Implementation is totally fungible.

However, this is based directly on the payload format in legacy desktop telemetry (which already made an appearance in Glean in the "old" timing distribution).  If there are advantages to retaining the same payload format, I'd say we leave this as-is, otherwise, I'm happy to simplify.

Reply:
Well one issue is that while this is _technically_ an array, it's really a tuple of length two, and we don't have a great way of dealing with tuples in BQ. Luckily because the elements are all the same type this ends up working out.